hrms/hr/doctype/attendance_request/attendance_request.py

```diff
@@ -118,7 +118,8 @@ def create_or_update_attendance(self, date: str):
 
 	def should_mark_attendance(self, attendance_date: str) -> bool:
 		# Check if attendance_date is a holiday
-		if is_holiday(self.employee, attendance_date):
+		if not self.ignore_holiday and is_holiday(self.employee, attendance_date):
```

Comment:
The Attendance Warnings dashboard should also reflect changes as per this checkbox. Currently it shows that attendance will be ignored on holiday dates

Reply:
@ruchamahabal I believe no changes are necessary in the message because if the checkbox is checked, the condition will not evaluate to true, and consequently, the message will not be displayed

Reply:
I am talking about this dashboard with warnings
https://frappehr.com/docs/v14/en/attendance-request#3-2-request-for-attendance-in-bulk

<img width="1363" alt="image" src="https://github.com/frappe/hrms/assets/24353136/472ff5a5-bde6-4857-abe9-790d7f504999">

https://github.com/frappe/hrms/blob/1bcc636b0108629ae547e6467c8aba8f90d2c72d/hrms/hr/doctype/attendance_request/attendance_request.py#L170

Reply:
@ruchamahabal I have a suggestion: before appending to the dashboard, consider checking whether the date corresponds to a holiday. This is important as the dashboard also displays on  leave data. it will avoid rewriting the warning. Please suggest

Reply:
Yes, go ahead. And add a test for this feature too